content/applications/finance/fiscal_localizations/belgium.rst

```diff
@@ -242,6 +242,208 @@ journal.
 
 .. _belgium/einvoicing:
 
+CodaBox
+-------
+
+**CodaBox** is a service that allows Belgian accounting firms accessing their clients' bank
+information and statements. Odoo provides a way to import such statements automatically.
+
+.. note::
+   As an accounting firm, you must manage your clients on separate databases and configure them
+   individually to avoid mixing up their data.
+
+Configuration
+~~~~~~~~~~~~~
+
+The configuration must be done on each client database. In the following
+instructions, we will refer to your client's company as *Company* and to your
+accounting firm as *Accounting Firm*.
+
+You must first :ref:`Install <general/install>` :guilabel:`CodaBox` and :guilabel:`CodaBox Bridge`
+to start.
+
+Configure the Company settings
+******************************
+
+#. Go to :menuselection:`Configuration --> Settings --> General Settings --> Companies`
+#. The country of the company must be set to Belgium
+#. The VAT number of the company must be filled
+#. The Accounting Firm and its VAT number must be filled (it can be left empty if the Accounting
+   Firm VAT number is the same as the Company VAT number)
+
+Configure the Journals
+**********************
+
+.. tabs::
+
+   .. tab:: For CODA files
+
+      #. Create a new bank journal
+      #. Set the right IBAN in the :guilabel:`Account Number`
+      #. Select :guilabel:`CodaBox synchronisation` as the :guilabel:`Bank Feed`
+
+      .. image:: belgium/codabox_configuration_coda_journal.png
+         :align: center
+         :alt: Configuration of a CODA journal.
+
+   .. tab:: For SODA files
+
+      #. Create a new miscellaneous journal
+      #. Go to :menuselection:`Accounting --> Configuration --> Accounting: CodaBox`
+      #. Select the journal you just created in the SODA journal field.
+
+      .. image:: belgium/codabox_configuration_soda_setting.png
+         :align: center
+         :alt: Configuration of a SODA journal.
+
+.. _belgium/codabox-configuration-connection:
+
+Configure the Connection
+************************
+
+First, you must select which users can access the **CodaBox** connection settings. To do so, go to
+:menuselection:`Settings --> Users & Companies --> Groups`, and search for the group
+**Allow to show and modify the CodaBox Connection Settings**. Add the users you want to this group.
+By default, only the **Administrator** is part of this group.
```

Comment:
:guilabel:`Allow to show and modify the CodaBox Connection Settings`. Add the users you want to this group.
By default, only the :guilabel:`Administrator` is part of this group.